tests/gsrpc/README.md

```diff
@@ -45,6 +46,15 @@ go run extrinsics/dataSubmitAndWatch/dataSubmitAndWatch.go -config config.json
 ```
 The same procedure of data submission is done here. But checks/logs if the data is included in the block. The default APP_ID is 0
 
+>Balance Transfer
+
+```
+go run extrinsics/balance/balance.go -config config.json
+```
+
+Amount is transferred from the seed account to the dest account with the amount mentioned in config file. Note that the dest account mentioned must be using the public hex key. 
+You can refer [here](https://docs.substrate.io/tutorials/build-a-blockchain/add-trusted-nodes/#generate-your-account-and-keys) on how to create or inspect account 
```

Comment:
Can we account ID instead? By using this function instead of hex:
https://pkg.go.dev/github.com/centrifuge/go-substrate-rpc-client@v2.0.0+incompatible/types#NewAddressFromAccountID

Reply:
yeh converted the account address to hexstring directly then used the same function